runhouse/main.py

```diff
@@ -139,9 +139,15 @@ def _start_server(
             f.readlines()  # Discard these, they're from the previous times the server was started
 
         # We do these one by one so it's more obvious where the error is if there is one
-        for cmd in cmds:
+        for i, cmd in enumerate(cmds):
+            last_cmd = i == len(cmds) - 1
             console.print(f"Executing `{cmd}`")
-            result = subprocess.run(shlex.split(cmd), text=True)
+            if (
+                last_cmd
```

Comment:
nit: can just do `if i == len(cmds) - 1` and avoid setting the `last_cmd` variable, I think it's pretty self explanatory what the expression means esp with the comment